src/translations/it.json

```
@@ -0,0 +1,1152 @@
+    "selectUser": "Seleziona utente",
+    "status": "{{active}} di {{total}} ruoli",
+    "tap_to_manage": "Tocca per gestire i ruoli",
+    "unassigned": "Non assegnato"
+  },
+  "settings": {
+    "about": "Informazioni",
+    "account": "Account",
+    "active_unit": "Unità attiva",
+    "app_info": "Informazioni app",
+    "app_name": "Nome app",
+    "arabic": "Arabo",
```

Biome (2.4.11)

[error] The key arabic was already declared.

(lint/suspicious/noDuplicateObjectKeys)